extensions/amp-consent/cmps/googlefc.md

```diff
@@ -0,0 +1,25 @@
+# Google Funding Choices
+
+## Example
+
+```html
+# This amp-consent extension will be generated by Funding Choices.
+<amp-consent id="consent" layout="nodisplay" type="googlefc">
+  <script type="application/json">
+    {
+      "clientConfig": {
+        "publisherIdentifier": "id"
+      }
+    }
+  </script>
+</amp-consent>
+```
+
+## Notes
+We are currently in a closed beta and have limited the usage of this amp-consent feature to a small batch of publishers. However, we are looking to open up access for general availability in the near future!
```

Comment:
1. Please move this section above Example
2. Please rename to something more specific than "Notes", like "Closed Beta".
3. Instead of "we", specify that it's Funding Choices. For example, "The `amp-consent` configuration for Funding Choices is in a closed Beta, and [...]"